HISTORY.md

```diff
@@ -1,5 +1,10 @@
 # Version History
 
+## 0.2.0_preview / 2021-04-19
+
+- Added OCS Assets models and API endpoints
+- Added type hints for all API models and endpoints
```

Comment:
There is considerable renaming of parameters and lots of reodering of parameters of functions (sometimes with additions, sometimes just reodered) this should be noted here

Reply:
Added a note, see if it works for you.